boards/shields/mikroe_ble_tiny_click/Kconfig.defconfig

```diff
@@ -3,4 +3,4 @@
 
 CONFIG_BT_HCI=y
 CONFIG_BT_HCI_ACL_FLOW_CONTROL=n
-CONFIG_BT_CTLR=n
+CONFIG_BT_LL_SW_SPLIT=n
```

Comment:
This is odd. We should not need to disable? Does this mean the shield supports Controller at SoC, but not on boards/sheilds?